src/docs/metrics/holders-distribution/index.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Holders Distribution
+author: Santiment Team
+date: 2020-08-24
+---
+
+## Definition
+
+There are five separate metrics regarding holders distribution:
+
+- holders_distribution_<interval> - represents the number of addresses holding the amount of tokens that belong to a given interval
```

Comment:
You can put the metric names in backticks so they look like this - `holders_distribution_<interval>`

Reply:
We also might want to have 2 lists of the available threhsolds and intervals, especially for the intervals as they're not very clear.